lib/crypto/packaged.py

```diff
@@ -45,13 +45,9 @@ def supports_firefox(file_obj):
             max__version_int__gte=version_int(settings.MIN_NOT_D2C_VERSION))
 
 
-def get_endpoint(file_obj):
-    """Get the endpoint to sign the file, depending on its review status."""
-    server = settings.SIGNING_SERVER
-    if (file_obj.status == amo.STATUS_BETA or
-            file_obj.version.addon.status != amo.STATUS_PUBLIC):
-        server = settings.PRELIMINARY_SIGNING_SERVER
-    if not server:
+def get_endpoint(server):
+    """Get the endpoint to sign the file, either the full or prelim one."""
+    if not server:  # Setting is empty, signing isn't enabled.
```

Comment:
is this the meat of it? I guess I was expecting a `try`/`except` block to catch signing errors?